### PR TITLE
upgrades

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

File: app/utils/d3-flame-graphs-v4/d3-flame-graph.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { scaleLinear, scaleQuantize } from 'd3-scale';
 import { min, max, range } from 'd3-array';
 import d3Tip from 'd3-tip';
 import { partition, hierarchy } from 'd3-hierarchy';
+import 'd3-transition';
```
Comment (rwjblue) [inline app/utils/d3-flame-graphs-v4/d3-flame-graph.js:6]: Why was this needed?

**Review by rwjblue — APPROVED** [review]

File: app/utils/d3-flame-graphs-v4/d3-flame-graph.js (hunk 1 above)
Comment (stefanpenner) [reply app/utils/d3-flame-graphs-v4/d3-flame-graph.js:6]: I believe it has side-affects, specifically adding `.transition` to the d3 object.
